src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1316,8 +1316,12 @@ def _validate_trusted_launch(namespace):
         return
 
     if not namespace.enable_vtpm or not namespace.enable_secure_boot:
-        logger.warning('It is recommended to specify "--enable-secure-boot True" and "--enable-secure-boot True"'
-                       ' to receive the full suite of security features that comes with Trusted Launch.')
+        logger.warning('It is recommended to specify "--enable-secure-boot True" and "--enable-vtpm True" to receive'
+                       ' the full suite of security features that comes with Trusted Launch. Breaking change:'
+                       ' --enable-secure-boot will be enabled by default in Microsoft Build and Ignite')
+
+    if not namespace.enable_vtpm:
+        namespace.enable_vtpm = True
```

Comment:
@sirfamelin In addition, do we need to print logs for the behavior that `vtpm` is enabled for TrustedLaunch by default, so that users can be aware of this default behavior

Reply:
Not required